opentracing-spring-web-itest/webflux/src/test/java/io/opentracing/contrib/spring/webflux/webfilter/itest/webflux/WebFluxJettyMinimalIntegrationTest.java

```diff
@@ -0,0 +1,112 @@
+
+    private static final ReactiveWebServerApplicationContext APPLICATION_CONTEXT = new ReactiveWebServerApplicationContext();
+    private static final MockTracer MOCK_TRACER = Mockito.spy(new MockTracer(new ThreadLocalScopeManager(),
+            MockTracer.Propagator.TEXT_MAP));
+    private final static TracingWebFilter TRACING_WEB_FILTER = new TracingWebFilter(
+            MOCK_TRACER,
+            Integer.MIN_VALUE,
+            Pattern.compile(""),
+            Collections.singletonList("/*"),
+            Arrays.asList(new WebFluxSpanDecorator.StandardTags(), new WebFluxSpanDecorator.WebFluxTags())
+    );
+    private static final int SERVER_PORT;
+    private static final TestRestTemplate TEST_REST_TEMPLATE;
+
+    static {
+        APPLICATION_CONTEXT.registerBean("jettyReactiveWebServerFactory", JettyReactiveWebServerFactory.class, () ->
+                new JettyReactiveWebServerFactory(0));
+        APPLICATION_CONTEXT.registerBean("webHandler", WebHandler.class, () ->
+                serverWebExchange -> TRACING_WEB_FILTER.filter(serverWebExchange, WebFluxJettyMinimalIntegrationTest::handler));
+        APPLICATION_CONTEXT.registerBean("httpHandler", HttpHandler.class, () ->
+                WebHttpHandlerBuilder.applicationContext(APPLICATION_CONTEXT).build());
+        APPLICATION_CONTEXT.refresh();
+        SERVER_PORT = APPLICATION_CONTEXT.getWebServer().getPort();
+        TEST_REST_TEMPLATE = new TestRestTemplate(new RestTemplateBuilder()
+                .rootUri("http://127.0.0.1:" + SERVER_PORT));
+    }
+
+    private static Mono<Void> handler(final ServerWebExchange serverWebExchange) {
+        final ServerHttpResponse response = serverWebExchange.getResponse();
+        response.setStatusCode(HttpStatus.OK);
+        return response.writeWith(Mono.just(new DefaultDataBufferFactory().wrap("Hello World!\n".getBytes())));
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        APPLICATION_CONTEXT.close();
+    }
+
+    @Before
+    public void beforeTest() {
+        MOCK_TRACER.reset();
+        Mockito.reset(MOCK_TRACER);
+    }
+
+    @Test
+    public void testGet() {
+        TEST_REST_TEMPLATE.getForEntity("/", String.class);
+
+        final List<MockSpan> mockSpans = MOCK_TRACER.finishedSpans();
+        Assert.assertEquals(1, mockSpans.size());
+
+        final MockSpan span = mockSpans.get(0);
+        Assert.assertEquals("GET", span.operationName());
+
+        Assert.assertEquals(8, span.tags().size());
+        Assert.assertEquals(Tags.SPAN_KIND_SERVER, span.tags().get(Tags.SPAN_KIND.getKey()));
+        Assert.assertEquals("GET", span.tags().get(Tags.HTTP_METHOD.getKey()));
+        Assert.assertEquals(TEST_REST_TEMPLATE.getRootUri() + "/", span.tags().get(Tags.HTTP_URL.getKey()));
+        Assert.assertEquals(200, span.tags().get(Tags.HTTP_STATUS.getKey()));
+        Assert.assertEquals("WebFlux/java-spring-web", span.tags().get(Tags.COMPONENT.getKey()));
```

Comment:
Change `java-spring-webflux` to be consistent with other component names in this repo?